docs/interchain-staking.md

```diff
@@ -0,0 +1,178 @@
+
+If and when the xStaking module contains complex delegations, the stakingKeeper method for calculating delegations would also need to include the Chain Registry reference to chain-id in order to properly calculate the delegation rewards with regard to the specific child chain. In the meantime, for Interchain Staking V1 this is unnecessary.
+
+
+## IBC & Cross Chain Validation
+
+There are a number of IBC application layer modules and packets that need to be developed to fully realize the IBC component of Interchain Staking. This work has begun with a spec draft from Informal Systems that is visible at [@informalsystems/cross-chain-validation](https://github.com/informalsystems/cross-chain-validation/). Instead of diving into the details of what they are and exactly how they work this section will be reserved for high level responsibilities of these mechanisms.
+
+There are three types of operations within Cross Chain Validation which must be present for Interchain Staking to take place:
+
+
+
+*   Validator Set Updates
+*   Reward Distribution
+*   Evidence
+
+
+### Validator Set Updates
+
+The primary duty of Cross Chain Validation is to relay the set of validators and their voting power. The inclusion of a validator within a set relevant to a specific child chain is designated within the Chain Registry. The voting power denominated in parent chain staking token is designated within the staking module. The xStaking module allows for the collation of validators and their delegations on a chain specific designation. This collation is what must be relayed to the child chain via Cross Chain Validation.
+
+The rate at which these validator set updates are relayed is a function of safety. At one extreme you could imagine the validator set being collated and relayed with every single block that is produced on the parent chain. This would ensure that the child chain has an absolutely accurate representation of validator weights at every potential state update within the parent chain. However, since delegations are subject to unbonding periods it is possible to approach state updates more conservatively. At another extreme, one may reason that if there is no active stake unbonding happening on the parent chain, it may be assumed that a validator set update will not be possible within a maximum duration of that unbonding period and so a validator set update can wait until just before that moment. Based on the possibility of instant redelegations this assumption may need to be further adjusted. 
```

Comment:
```suggestion
The rate at which these validator set updates are relayed is a function of safety. At one extreme you could imagine the validator set being collated and relayed with every single epoch that is produced on the parent chain. This would ensure that the child chain has an absolutely accurate representation of validator weights at every potential state update within the parent chain. However, since delegations are subject to unbonding periods it is possible to approach state updates more conservatively. At another extreme, one may reason that if there is no active stake unbonding happening on the parent chain, it may be assumed that a validator set update will not be possible within a maximum duration of that unbonding period and so a validator set update can wait until just before that moment. Based on the possibility of instant redelegations this assumption may need to be further adjusted. 
```